testbeam_analysis/dut_alignment.py

```diff
@@ -708,7 +708,7 @@ def apply_alignment(input_hit_file, input_alignment_file, output_hit_file, inver
         Filename of the input alignment file.
     output_hit_file : string
         Filename of the output hits file with hit data after alignment was applied.
-    inverse : boolean
+    inverse : bool
```

Comment:
This introduces a regression here: https://github.com/SiLab-Bonn/testbeam_analysis/blob/99f8ab255e2207436cac56bab3770550f1178ec8/testbeam_analysis/gui/analysis_widgets.py#L328

Reply:
This is more of an issue for the gui branch than development?

Reply:
but ```bool``` is the type name and ```boolean```should be avoided?